packages/ndla-icons/src/Icon.tsx

```diff
@@ -7,48 +7,82 @@
  */
 
 import { ReactNode, SVGAttributes, useMemo } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
 
 export interface Props extends SVGAttributes<SVGSVGElement> {
   title?: string;
   description?: string;
   children?: ReactNode;
-  size?: string;
+  size?: keyof typeof sizes;
   ariaHidden?: boolean;
 }
+
+const StyledIcon = styled.svg`
+  display: inline-block;
+  fill: currentColor;
+  vertical-align: middle;
+  line-height: 1em;
+  flex-shrink: 0;
+`;
+
+const sizes = {
+  xsmall: css`
+    width: ${spacing.xsmall};
+    height: ${spacing.xsmall};
+  `,
+  small: css`
+    width: ${spacing.small};
+    height: ${spacing.small};
+  `,
+  normal: css`
+    width: ${spacing.normal};
+    height: ${spacing.normal};
+  `,
+  nsmall: css`
+    width: ${spacing.nsmall};
+    height: ${spacing.nsmall};
+  `,
```

Comment:
Pirk: naturlig rekkefølge
```suggestion
  nsmall: css`
    width: ${spacing.nsmall};
    height: ${spacing.nsmall};
  `,
  normal: css`
    width: ${spacing.normal};
    height: ${spacing.normal};
  `,
```